modules/rostests/apitests/crt/math_helpers.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Helpers for testing math functions
+ * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#pragma once
+
+#define _USE_MATH_DEFINES
+#include <math.h>
+#include <float.h>
+#include <apitest.h>
+
+static
+__inline
```

Comment:
Any reason on using `__inline` instead of `inline`?

Reply:
`inline` is C++. `__inline` is C.

Reply:
`inline` is also C

Reply:
https://stackoverflow.com/a/2765211
> `inline` is the keyword, in C++ and C99.
>
> `__inline is a vendor-specific keyword (e.g. [MSVC](https://msdn.microsoft.com/en-us/library/bw1hbe6y.aspx)) for inline function in C, since C89 doesn't have it.